pythonx/UltiSnips/snippet_manager.py

```diff
@@ -882,8 +883,6 @@ def _file_to_edit(self, requested_ft, bang):
             # the users vim config directory.
             dot_vim_dir = vim_helper.get_dot_vim()
             for snippet_dir in all_snippet_directories:
-                if Path(dot_vim_dir) != Path(snippet_dir).parent:
```

Comment:
This was introduced in #1138. I think it was meant so that if there is a HOME dir for the user, it should be used for editing snippets preferably. You disable this, which makes sense for Windows, but not for most other OSes.